### ISSUE TITLE
gnt-instance info fails with exception related to netinfo

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#434]
Originally reported of Google Code with ID 378.

```
Invoking gnt-instance info myinstance 
results in an exception:

[gnt-dev-helgav.hot] root@wpeg16:~# gnt-instance info myinstance
Traceback (most recent call last):
  File "/usr/sbin/gnt-instance", line 21, in <module>
    sys.exit(main.Main())
  File "/usr/lib/python2.6/dist-packages/ganeti/client/gnt_instance.py", line 1602, in Main
    env_override=_ENV_OVERRIDE)
  File "/usr/lib/python2.6/dist-packages/ganeti/cli.py", line 2399, in GenericMain
    result = func(options, args)
  File "/usr/lib/python2.6/dist-packages/ganeti/client/gnt_instance.py", line 1183, in ShowInstanceConfig
    (idx, mac, ip, mode, link, netinfo["name"]))
```
Originally added on 2013-02-19 13:01:40 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-02-19 13:02:01 +0000 UTC.

Added Labels: Type-Defect

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with:
commit c8f4e4256014033c351f3cda857068e84b6f0656
Author: Dimitris Aragiorgis <dimara@grnet.gr>
Date:   Tue Feb 19 16:45:30 2013 +0200

    Fix issue 378
    
    In case a NIC is not inside a network then netinfo None. Thus
    netinfo["name"] fails.
    
    Signed-off-by: Dimitris Aragiorgis <dimara@grnet.gr>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2013-02-19 16:58:45 +0000 UTC.

Mentioned Issue with GoogleCode ID 378 has been migrated to GitHub with ID #434.
Changed State: Fixed
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
